src/app/workflow/info-tab/info-tab.component.html

```diff
@@ -416,7 +416,14 @@
             </div>
           </li>
           <div>
-            <strong matTooltip="Digital Object Identifier for all workflow versions">DOI</strong>:
+            <strong matTooltip="Digital Object Identifier for all workflow versions">Concept DOI</strong>
+            <img
+              matTooltip="This DOI represents all versions, and will always resolve to the latest one"
+              src="../../../../assets/svg/help.svg"
+              alt="help"
+              class="doi-help-icon"
```

Comment:
I believe the convention should be to use `.pl-2` instead. Instead of introducing a new class that duplicates an existing one. @kathy-t can confirm.

Reply:
Yes, I recommend using `pl-2` to avoid duplication https://github.com/dockstore/dockstore-ui2/blob/b4a52e7864b579354c79a12f28d732d922082384/src/bootstrap4.scss#L280-L283